src/Sylius/Behat/Element/Admin/Product/ProductAssociationsFormElement.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Element\Admin\Product;
+
+use Behat\Mink\Session;
+use Sylius\Behat\Service\DriverHelper;
+use Sylius\Behat\Service\Helper\AutocompleteHelperInterface;
+use Sylius\Component\Core\Model\ProductInterface;
+use Sylius\Behat\Element\Admin\Crud\FormElement as BaseFormElement;
+use Sylius\Component\Product\Model\ProductAssociationTypeInterface;
+
+final class ProductAssociationsFormElement extends BaseFormElement implements ProductAssociationsFormElementInterface
```

Comment:
As these form elements are in the `Product` directory, I would be for removing the `Product` suffix from them:
```suggestion
final class AssociationsFormElement extends BaseFormElement implements AssociationsFormElementInterface
```
Wdyt? 🤔